corehq/apps/data_cleaning/models.py

```diff
@@ -902,6 +902,19 @@ def create_session_defaults(self, session):
                 is_system=self.model.is_system_property(prop_id),
             )
 
+    def create_for_session(self, session, prop_id, label, data_type=None):
+        is_system_property = self.model.is_system_property(prop_id)
+        from corehq.apps.data_cleaning.utils.cases import get_system_property_data_type
```

Comment:
Another local import that might be worth looking at

Reply:
added to followup list